src/Arrays/Programs_involving_Data_Sequences/Hard/Q1/CountArTest.java

```diff
@@ -0,0 +1,73 @@
+package Arrays.Programs_involving_Data_Sequences.Hard.Q1;
+
+import global.BaseTest;
+import global.variables.*;
+import org.junit.jupiter.api.RepeatedTest;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CountArTest extends BaseTest{
+    public Clause[] testSentence() {
+        return new Clause[]{
+                new IntegerLiteral(0),
+                new NewLine(),
+                new IntegerLiteral(1),
+                new NewLine(),
+                new IntegerLiteral(0),
+                new NewLine(),
+                new IntegerLiteral(2),
+                new NewLine(),
+                new IntegerLiteral(3),
+                new NewLine(),
+                new IntegerLiteral(1),
+                new NewLine(),
+                new IntegerLiteral(0),
+                new NewLine(),
+                new IntegerLiteral(2),
+                new NewLine(),
+                new IntegerLiteral(0),
+                new NewLine(),
+                new IntegerLiteral(3),
```

Comment:
try looping this as our previous one, might save time hardcoding this. Also the variables for IntegerLiteral should be defined outside of clause as int[] or multiple int and pasted into IntegerLiteral.

Reply:
Yeah, I would just change this to a loop, but otherwise looks good